src/mixins/view.js

```diff
@@ -205,7 +206,7 @@ const ViewMixin = {
   triggerMethod() {
     const ret = triggerMethod.apply(this, arguments);
 
-    this._triggerEventOnBehaviors(arguments);
+    this._triggerEventOnBehaviors.apply(this, arguments);
```

Comment:
I tried a few times to see a perf difference between this and the other.. couldn't really find a reason why these two `_trigger` functions should be different?